docs/how-to-guides/run-chainhook-as-a-service-using-bitcoind.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Run Chainhook as a Service using Bitcoind
```

Comment:
I'm curious about these titles. The way the render in the markdown is kind of strange. Do we need these for the website?

Reply:
I removed the H1 to avoid duplicate titles on the rendered markdown page.